src/cmd/run.js

```diff
@@ -111,13 +182,14 @@ export default function run(
         profilePath: firefoxProfile,
       });
     })
-    .then((runner) => {
-      return runner.getProfile().then((profile) => {
+    .then((runner: ExtensionRunner) => {
+      return runner.getProfile().then((profile: FirefoxProfile) => {
         return {runner, profile};
       });
     })
     .then((config) => {
-      const {runner, profile} = config;
+      const runner: ExtensionRunner = config.runner;
```

Comment:
are you sure this type hint is necessary? I thought Flow would figure it out because it would be tracing the `runner` instance from https://github.com/rpl/web-ext/blob/d5083f42002b2dcb5e72f8c886997d0035c1c0c7/src/cmd/run.js#L177